packages/editor-ui/src/App.vue

```diff
@@ -9,14 +9,16 @@
 				[$style.sidebarCollapsed]: sidebarMenuCollapsed
 			}"
 		>
-			<div id="header" :class="$style['header']">
+			<div id="header" :class="$style.header">
 				<router-view name="header"></router-view>
 			</div>
-			<div id="sidebar" :class="$style['sidebar']">
+			<div id="sidebar" :class="$style.sidebar">
 				<router-view name="sidebar"></router-view>
 			</div>
-			<div id="content" :class="$style['content']">
-				<router-view />
+			<div id="content" :class="$style.content">
+				<keep-alive include="NodeView" :max="1">
```

Comment:
why is this needed? could this have memory or perf implications?

Reply:
This is keeping the main `NodeView` active, so it's not reloading every time we switch tabs. I guess this is the UX we are after in this feature. 
`include` and `max` parameters will limit router view so it only caches one instance of `NodeView`and not other components that render in that slot.